tests/unit/src/test/resources/workspace-symbol/example/MacroAnnotation.scala

```diff
@@ -14,12 +14,12 @@ object MacroAnnotations/*example.MacroAnnotations.*/ {
   import scala.meta._
   // IntelliJ has never managed to goto definition for the inner classes from Trees.scala
   // due to the macro annotations.
-  val x: Defn.Class = Defn.Class(
+  val x/*example.MacroAnnotations.x.*/: Defn.Class = Defn.Class(
```

Comment:
Do we have examples in the tests with local symbols? If not I would add some to make sure we don't show local stuff

Reply:
Yes, we do.